core/assets/bundles/bundle_es.properties

```diff
@@ -1975,7 +1975,7 @@ block.electric-heater.description = Calienta los bloques a los que apunta. Requi
 block.slag-heater.description = Calienta los bloques a los que apunta. Requiere magma.
 block.phase-heater.description = Calienta los bloques a los que apunta. Requiere tejido de fase.
 block.heat-redirector.description = Redirige el calor que acumula a otros bloques.
-block.heat-router.description = Spreads accumulated heat in three output directions.
+block.heat-router.description = Divide el calor acumulado en tres direcciones de salida.
```

Comment:
This should use `térmico` instead of `calor`

Reply:
Yeah you are right, I was thinking more of not using ``calor`` but it doesn't make sense.
It seems it doesnt let me resolve the change lol